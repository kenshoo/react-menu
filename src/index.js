export { Menu } from "./components/menu";
export { Button } from "./components/button/button";
export { RightMenu } from "./components/right_menu/right_menu";
export { LeftMenu } from "./components/left_menu/left_menu";
