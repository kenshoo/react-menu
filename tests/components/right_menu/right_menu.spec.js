import React from "react";
import renderer from "react-test-renderer";

import { RightMenu } from "../../../src/";

describe("RightMenu component", () => {
  test("renders children", () => {
    const cmp = renderer.create(<RightMenu>TEST</RightMenu>);
    expect(cmp).toMatchSnapshot();
  });

  test("recieves className", () => {
    const cmp = renderer.create(
      <RightMenu className="TEST_CLASS">TEST</RightMenu>
    );
    expect(cmp).toMatchSnapshot();
  });
});
