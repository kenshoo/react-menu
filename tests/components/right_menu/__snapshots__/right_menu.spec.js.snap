// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RightMenu component recieves className 1`] = `
<div
  className="right_menu TEST_CLASS"
>
  TEST
</div>
`;

exports[`RightMenu component renders children 1`] = `
<div
  className="right_menu"
>
  TEST
</div>
`;
